gruntfile.js

```diff
@@ -134,7 +158,8 @@ module.exports = function (grunt) {
       'npm-contributors',
       'bump:' + (type || 'patch') + ':bump-only',
       'build',
-      'changelog',
+      'conventionalChangelog',
+      'conventionalGithubReleaser',
```

Comment:
This shouldn't matter if `allBlocks = true`. `conventionalGithubReleaser` should work exactly the same way as `conventionalChangelog`. `allBlocks = true` here because we want to regenerate all previous ones (once this has run once we can remove it). Once `allBlocks = true` is removed we actually wanna run it between changing the version in package.json, and commit & tag, just like `conventionalChangelog`